### ISSUE TITLE
Media Gallery with small images in CB v3

### ISSUE THREAD

**Issue by Koxae** [htmlburger/carbon-fields#677]
## Version

  - Carbon Fields: 3.0.2
  - WordPress: 5.1
  - PHP: 5.6, 7, 7.1

## Expected Behavior
Adding any size images to the Media Gallery.

## Actual Behavior
If I try to add a small image to the Media Gallery, then the container with the Media Gallery instantly disappears from the page code.

## Container definition
```php
add_action( 'carbon_fields_register_fields', 'cg_theme_options' );
function cg_theme_options()
{
$basic_options_container = Container::make( 'theme_options', __( 'Theme Options' ) )
		 ->add_fields( array(
			Field::make( 'text', 'cg_tw', __( 'TW' ) ),
			Field::make( 'media_gallery', 'clients', __( 'Clients' ) )
			->set_type( 'image' )
			->set_duplicates_allowed( false ),
		 ) );
}
```

## Steps to Reproduce the Problem

  1. Click on Select Attachments in the Media Gallery
  2. Select a small image in the list of media files.
  3. Click confirmation button

![screenshot-2019 02 27-16-56-48](https://user-images.githubusercontent.com/36166503/53497224-c844ff00-3ab4-11e9-820a-0ed1a8187087.png)

## Comments
I replaced the file Media_Gallery_Fields.php with an older version. The error remained.

**Comment by Dimox** [comment]
I confirm this problem. And console shows the following error:

```
vendor.min.js?ver=3.0.2:49 TypeError: Cannot read property 'url' of undefined
    at core.min.js?ver=3.0.2:1
    at Array.map (<anonymous>)
    at core.min.js?ver=3.0.2:1
    at core.min.js?ver=3.0.2:1
    at ai (vendor.min.js?ver=3.0.2:49)
    at vendor.min.js?ver=3.0.2:49
    at Ki (vendor.min.js?ver=3.0.2:49)
    at qi (vendor.min.js?ver=3.0.2:49)
    at Au (vendor.min.js?ver=3.0.2:49)
    at Pu (vendor.min.js?ver=3.0.2:49)
```

**Comment by wantToHawaii** [comment]
I have the same issue!

**Comment by EMembrillas** [comment]
Any solution for this problem?

**Comment by lstoyanoff** [comment]
Hi, this issue was fixed in 3.1.3
